blog/2024/deep-dive-into-amm-integration.md

```diff
@@ -0,0 +1,114 @@
+
+## Core Features of the AMM Integration on XRPL
+
+**Protocol Native:**
+
+* As a core primitive of the XRPL, developers can utilize AMM functionality without the need to create their own smart contracts and face associated risks.
+
+**Aggregated Liquidity:**
+
+* Liquidity for all trading pairs is aggregated at the protocol layer, rather than fragmented or siloed across individual smart contracts. 
+
+**Continuous Auction Mechanism:**
+
+* The implementation of a continuous auction mechanism allows arbitrageurs to bid to capture price discrepancies at a discount.
+* The AMM continuously auctions trading advantages for 24-hour periods at near zero trading fees. This results in immediate arb trading and maintains stable volatility.
+* Proceeds from each auction are partially refunded to the prior arbitrage slot holder and partially burnt, effectively reducing impermanent loss for liquidity providers.
+
+**Single-sided Liquidity Provision:**
+
+* Only a single asset is required to contribute to a pool, decreasing the required number of steps for users to contribute liquidity.
+
+**No Miner Extractable Value (MEV):**
+
+* The XRPL uses federated consensus, meaning that each ledger (equivalent to a “block”) is determined by a consensus of participants whereby no single party can dictate which transactions or ledgers are valid. There are no miners to prioritize only certain transactions (namely higher gas fee orders) to the Ledger.
+* The AMM’s (and existing DEX) orders inherit the core ledger’s deterministic-random transaction ordering secured by the distributed network of validators. While it does not eliminate the ability to front run transactions, it does become more difficult compared to an open mem pool.
+
+**CLOB DEX Integration:**
+
+* The AMM is integrated with the central limit order book (CLOB)-based DEX and enables price optimization to determine whether swapping within a liquidity pool, through the order book, or both provides the best rate and executes accordingly. This ensures that transactions can automatically use the most efficient path for trades, whether through offers on the DEX or through AMM pools, optimizing for the best possible exchange rates thereby enhancing DEX liquidity.
+
+## Core Transactions of XRPL AMM Protocol
+
+* **AMMCreate:** Enables the creation of a new AMM instance, initializing it with funding and defining the asset pair.
+* **AMMDeposit:** Allows users to add liquidity to an existing AMM, receiving LP (Liquidity Provider) Tokens in proportion to their contribution.
```

Comment:
...receiving Liquidity Provider (LP)  tokens in proportion to their contribution.